pkg/parser/errorid/errorid.go

```diff
@@ -56,6 +56,11 @@ var matchers = []matcher{
 	{id: "use_of_uninitialized_value", substrings: []string{"use-of-uninitialized-value"}},
 
 	// more global issues, should be at the end so they do not overwrite more explicit ones
+	{
+		id:         "undefined behavior",
```

Comment:
```suggestion
		id:         "undefined_behavior",
```
all the other IDs are like this